scripts/generate_pull_request_pipelineruns.py

```
@@ -0,0 +1,237 @@
+        component=component,
+        dockerfile=pathlib.Path(next(param for param in push_pipeline['spec']['params']
+                                     if param['name'] == 'dockerfile')['value']),
+        build_platforms=build_platforms,
+    )
+
+    # Generate the new filename and write the file
+    pr_pipeline_path = pathlib.Path(str(push_pipeline_path).replace(PUSH_SUFFIX, PR_SUFFIX))
+
+    try:
+        with open(pr_pipeline_path, 'w') as f:
+            print(f"# yamllint disable-file", file=f)
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)